doc/user_guide/encoding.rst

```diff
@@ -349,4 +349,83 @@ Shorthand            Equivalent long-form
 ``x='sum(name)'``    ``alt.X('name', aggregate='sum')``
 ``x='sum(name):Q'``  ``alt.X('name', aggregate='sum', type='quantitative')``
 ``x='count():Q'``    ``alt.X(aggregate='count', type='quantitative')``
+``order='name'``     ``alt.Order('name')``
```

Comment:
This table is about translation of shorthand form to full arguments, so adding order here is not really necessary.